Gesture_control.py

```diff
@@ -96,6 +117,7 @@ def detect_motion(x1,y1,x2,y2,t):
     else:
         return NO_MOTION
 
+
 # performing actions based on hand motion
 def performAction(hand_motion):
```

Comment:
Follow the snake case convention that we have been following for the function names.
make it perform_action